p2panda-rs/src/operation/mod.rs

```diff
@@ -117,7 +117,7 @@
 //!                                                 │
 //!                                                 ▼
 //!                                         ┌─────────────────┐
-//!                                         │VerifiedOperation│
+//!                                         │PublishedOperation│
```

Comment:
`PublishedOperation` is a type only used in `test_utils`, right? I'd not mention it here when its not part of the public API.

Reply:
Ah yes, this was an accident ;-p